tests/test_transaction.py

```diff
@@ -100,7 +111,8 @@ def test_no_input(self):
     def test_message(self):
         unspents_original = [Unspent(10000, 0, '', '', 0),
                              Unspent(10000, 0, '', '', 0)]
-        outputs_original = [(BITCOIN_CASHADDRESS_TEST_COMPRESSED, 1000, 'satoshi')]
+        # outputs_original = [(BITCOIN_CASHADDRESS_TEST_COMPRESSED, 1000, 'satoshi')]
```

Comment:
Why are all of these commented out regarding the TEST_ part?

Reply:
I forgot to remove them. Fixed [https://github.com/pybitcash/bitcash/pull/82/commits/5bbaa343501b3242910f1fed18606b5d9a8b920c](https://github.com/pybitcash/bitcash/pull/82/commits/5bbaa343501b3242910f1fed18606b5d9a8b920c)